doc/reposync.rst

```diff
@@ -46,7 +46,11 @@ All general DNF options are accepted. Namely, the ``--repoid`` option can be use
     Delete local packages no longer present in repository.
 
 ``--download-metadata``
-    Download all repository metadata. Downloaded copy is instantly usable as a repository, no need to run createrepo_c on it.
+    Download all repository metadata. Downloaded copy is instantly usable as a repository, no need to run createrepo_c
+    on it. When the option is used with `--newest-only`, only latest packages will be downloaded, but metadata will
+    still contain older packages. When DNF will try to download older packages it ends with an error, due to missing
+    file. It might by useful to update metadata using `createrepo_c --update` to remove packages with missing RPM file
+    from metadata.
```

Comment:
The last two sentences are a bit clunky. How about to swap them: "It might by useful to update the metadata using `createrepo_c --update` to remove the packages with missing RPM files from metadata. Otherwise, DNF ends with an error due to the missing files whenever it tries to download older packages."

Reply:
Also `might by` -> `might be`.

Reply:
Thank you for great suggestion. I only did not use word `metadata` together with `the`.